views/index.html

```diff
@@ -161,7 +161,51 @@ <h2 class="about-header">Skills</h2>
                 </div>
             </article>
         </section>
+
     </main>
+    <footer id="footer">
+        <img src="../images/image_geometry_1.png" alt="footer image">
+        <section class="intro">
+            <h2 class="name">Get started</h2>
+            <p class="description">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+                incididunt ut labore et dolore magna.</p>
+                <form  action="https://formspree.io/f/01">
+                    <ul class="info">
```

Comment:
- [ ] Please, consider that you should set the method with POST value. Your form gives an error after clicking the button. 
<img width="291" alt="Screen Shot 2022-01-12 at 01 34 05" src="https://user-images.githubusercontent.com/49064106/149025383-56541710-29a0-4224-9fe1-a1a868bc47ce.png">